web/src/settings_components.ts

```diff
@@ -860,7 +860,8 @@ export function check_stream_settings_property_changed(
     const current_val = get_stream_settings_property_value(property_name, sub);
     let proposed_val;
     switch (property_name) {
-        case "can_remove_subscribers_group": {
+        case "can_remove_subscribers_group":
+        case "can_administer_channel_group": {
             const pill_widget = get_group_setting_widget(property_name);
             assert(pill_widget !== null);
             proposed_val = get_group_setting_widget_value(pill_widget);
```

Comment:
This list can also likely be replaced with accessing the list of groups.

Reply:
Fixed in #32558 (2nd commit)